core/types/src/privacy.ts

```diff
@@ -0,0 +1,21 @@
+export type Circuit = {
+  id: string;
+  name: string;
+  description?: string;
+  type?: string;
+  provider: string;
+  curve?: string;
+  constraintSystem?: string;
+  metadata?: object;
+}
+
+export type Commitment = {
+  metadata: object; // arbitrary data
+  location: number;
+  salt: string; // salt such that currentHash = H(data + salt)
+  proof: number[];
+  publicInputs: string[];
+  sender: string; // address of participant who created the commitment
```

Comment:
Which address is this meant to be? The zkp public key? Would be good to update the comment to clarify

Reply:
This matches the `Participant` address.